docs/index.md

```
@@ -1,13 +1,13 @@
 hide:
-    - navigation 
+    - navigation
 
 # Introduction
 
 <img align="left" src="logo.png">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)